### PR TITLE
feat (dbt): allow syncing only exposures

### PR DESCRIPTION
**Pull request by betodealmeida**
Allow syncing only exposures from Preset workspaces to a YAML file.

This now works with both dbt Cloud and dbt Core:

```bash
$ preset-cli superset sync dbt-core ... --exposures=exposures.yaml --exposures-only
$ preset-cli superset sync dbt-cloud ... --exposures=exposures.yaml --exposures-only
```

The datasets don't have to be created via the CLI for the exposure discovery to work. The command will look at all models (filtered by `--select` and `--excluded`, if set) and figure out which datasets correspond to them based on the the database, schema, and table name.

### CODE REVIEW

**Comment by betodealmeida** [comment]
This closes #142

**Review by anupreet — APPROVED** [review]
based on your tests and dbt documentation, the code LGTM

File: src/preset_cli/cli/superset/sync/dbt/command.py [hunk 1]
```diff
@@ -75,6 +81,7 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-locals
     import_db: bool = False,
     disallow_edits: bool = True,
     external_url_prefix: str = "",
+    exposures_only: bool = True,
```
Comment (eschutho) [inline src/preset_cli/cli/superset/sync/dbt/command.py:84]: since you have this defaulting to False above, do you want the default here to match?

Comment (betodealmeida) [reply src/preset_cli/cli/superset/sync/dbt/command.py:84]: Oh, good catch!

Comment (betodealmeida) [reply src/preset_cli/cli/superset/sync/dbt/command.py]: It doesn't really matter, but it's good to be consistent. Updated it.

**Pull request merged**